src/styles/main.scss

```diff
@@ -1,3 +1,208 @@
-body {
-  margin: 0;
+$color-main: #060b35;
+$color-secondary: #616070;
+$border-radius: 5px;
+$border-card: 1px solid #f3f3f3;
+$blue-color: #00acdc;
+$white-color: #fff;
+$button-border-hover: 1px solid #00acdc;
```

Comment:
Create a separate file for variables

Reply:
Please check, all changes were done in accordance with your comments.